publisher/rootfs/Dockerfile

```diff
@@ -4,7 +4,8 @@ FROM alpine:3.1
 # profiling information without any additional package installation.
 RUN apk add --update-cache curl && rm -rf /var/cache/apk/*
 
-ADD bin/publisher /usr/local/bin/publisher
-ENTRYPOINT ["/usr/local/bin/publisher"]
+COPY . /
```

Comment:
Yes! `COPY` is usually preferable to `ADD`, we should be making this change everywhere as we revisit Dockerfiles.